cps/tasks/download.py

```diff
@@ -81,7 +81,7 @@ def run(self, worker_thread):
                     else:
                         elapsed_time = (datetime.now() - last_progress_time).total_seconds()
                         if elapsed_time >= fragment_stuck_timeout:
-                            self.message += f"<br>Some fragments are taking longer than expected to download. Please wait..."
+                            self.message = f"Unable to download {self.media_url_link} due to unavailable fragments. Please see https://github.com/yt-dlp/yt-dlp/issues/2137 for context. Try to download the video again later."
```

Comment:
> Try to download the video again later.

Specifically why?  Under what conditions will waiting help?

```suggestion
                            self.message = f"Unable to download {self.media_url_link} due to unavailable fragments. See https://github.com/yt-dlp/yt-dlp/issues/2137 for context. Try to download the video again later."
```

Reply:
> > Try to download the video again later.
> 
> Specifically why? Under what conditions will waiting help?

Let's not create false hopes:

- Outline conditions (here on #223 discussion) where waiting can help.
- If these are pragmatic and real, that this too should be summarized within "Tasks" view